themes/aframe/layout/index.ejs

```diff
@@ -35,6 +35,6 @@
       Get Started
     </a>
 
-    <a class="credits borderless-link" href="http://mozvr.com/" rel="external"></a>
+    <div class="sponsor"><div class="message">Supported by</div><a class="mozilla-logo" href="http://mozilla.com/" rel="external"></a></div>
```

Comment:
I'd point to `https://www.mozilla.org/` instead of `http://mozilla.com/` (to save a few redirects)